app/models/transfer.rb

```diff
@@ -48,6 +48,11 @@ def to_path
     end
   }
 
+  def self.latest_transfers(patients)
+    tuples = patients.pluck(:id, :latest_transfer_at)
+    Transfer.where("(patient_id, created_at) IN (#{tuples.collect { '(?, ?)' }.join(', ')})", *tuples.flatten)
```

Comment:
This string causes a brakeman warning. If there is a way we can avoid the warning altogether that would be great, otherwise I think it will need to be ignored.